src/Config/Auth.php

```diff
@@ -127,6 +127,17 @@ class Auth extends BaseConfig
         'tokens',
     ];
 
+    /**
+     * --------------------------------------------------------------------
+     * Login validation rules
```

Comment:
This would make more sense down next to `$validFields` since they influence each other.